src/config/graphical-session/kde.md

```diff
@@ -17,3 +17,23 @@ instead.
 If you wish to start an X-based session from the console, use
 [startx](./xorg.md#startx) to run `startplasma-x11`. For a Wayland-based
 session, run `startplasma-wayland` directly.
+
+## Dolphin
+
+Dolphin is the default file manager of the KDE desktop environment. It can be
+installed on its own by installing the `dolphin` package, or it can be installed
+as part of the `kde5` or `kde5-baseapps` packages.
+
+### Thumbnail Previews
+
+To have properly working thumbnail file previews, the packages
+`ffmpegthumbnailer` and `kdegraphics-thumbnailers` have to be installed. Enable
+previews in "Control" -> "Configure Dolphin" -> "General" -> "Previews" by
+checking the corresponding boxes. File previews will be shown for the selected
+file types after clicking "Preview" in dolphin's toolbar.
```

Comment:
I guess dolphin should be capitalized? So "in Dolphin's toolbar".

Reply:
I've capitalized it.